components/schema-migrator/validate.sh

```diff
@@ -92,7 +123,10 @@ function migrationProcess() {
 
     echo -e "${GREEN}Migrations for \"${db}\" database and \"${path}\" path${NC}"
     migrationUP "${path}" "${db}"
-    migrationDOWN "${path}" "${db}"
+
+    if [[ ! -f seeds/dump.sql ]]; then
```

Comment:
Why don't we execute down migrations if we've used a db dump? Is that ok?

Reply:
There are some conceptual things to be agreed upon for this to work. Let's discuss it.

Reply:
Discussed with team.